### DISCUSSION TITLE
作者，能不能添加“提示音频“？

### DISCUSSION THREAD

**Discussion by skyfamster** [fishaudio/Bert-VITS2#251] [Ideas]
像现在的emotion提示词一样，但是是具体的同样文字的音频：根据“提示音频”的语速，高低音作为标准尽量靠拢，生成音频。既有模型说话的特质，又有想要的感情。
现在生成的东西没有重音，也没有变化。假如能做出来的话，可以对配音表演行业有巨大的推进。

**Comment by Stardust-minus** [comment]
> 像现在的emotion提示词一样，但是是具体的同样文字的音频：根据“提示音频”的语速，高低音作为标准尽量靠拢，生成音频。既有模型说话的特质，又有想要的感情。 现在生成的东西没有重音，也没有变化。假如能做出来的话，可以对配音表演行业有巨大的推进。

可以尝试训练先前带有emotion的版本，或者期待一下fish-speech吧~

**Reply by skyfamster** [reply]
我要的像是so vits 和rvc这类功能，但这俩我都试了，转的结果没有这个bert的好，这个bert生成的音频可以保留模型的一些特点，比如口音
；所以我希望这个BERT能添加个提示音频对最终结果产生影响“具体的该在哪里停顿，该在哪里加重语气什么的” 这类具体的，根据这个作为基础，包裹上模型音色像是另外一个人说话。

**Comment by skyfamster** [comment]
感谢作者组花时间钻研分享，我会继续关注

**Comment by litagin02** [comment]
Maybe you can try my [Style-Bert-VITS2](https://github.com/litagin02/Style-Bert-VITS2), which is a fork based on Bert-VITS2 v2.1 with more controllable style.

This replaces emo embeddings of v2.1 to another one, remove vector quantizer and just fully-connected layer, but user must create style vectors for each model manually, which can (and all other features) be done in WebUI.

For now all are written in Japanese, sorry.

**Reply by skyfamster** [reply]
No.I just want to be able to add my custom audio file as an effector,
 which impacts the result by“rythme and stress",but the voice tone(accent) is still model's.
bert vits style's “rythme and stress" is automatically generated by machine, I'd want to control more precisely. 
what makes bert 's result better than RVC and SOVITS is there's text tag during the training  process ,if there's a way to combine the 2 project,that'll be right!

**Reply by litagin02** [reply]
Sorry I just misunderstood that you want more style or emotion control. I agree we want more explicit precise rhythm and stress and accent control. I hope someone will develop such project.
(I think RVC and sovits has VITS in its encoder, so we may replace VITS part of RVC with VITS2 or Bert one, but I'm not specialist of RVC so I cannot say anymore.)

**Comment by wertyac** [comment]
> 像现在的emotion提示词一样，但是是具体的同样文字的音频：根据“提示音频”的语速，高低音作为标准尽量靠拢，生成音频。既有模型说话的特质，又有想要的感情。 现在生成的东西没有重音，也没有变化。假如能做出来的话，可以对配音表演行业有巨大的推进。

问一下你用的是中文加强版吗？我测试emotion感觉没有效果。emotion感觉应该用英文做为prompt。clap模型应该只支持英文的。

**Reply by skyfamster** [reply]
我沒用中文加强，里边有EMOTION，文字提示可以是中文也可以英语，有效果的，但远达不到剧配音的要求，我要的是rvc的那种
